tests/phpunit/src/Application/KernelTest.php

```diff
@@ -77,6 +77,8 @@ private function getEnd(): string {
   ide:list:app             [ide:list] List available Cloud IDEs belonging to a given application
   ide:list:mine            List Cloud IDEs belonging to you
   ide:open                 Open a Cloud IDE in your browser
+ org
+  org:member-remove        Remove the members from an organiztaion on Acquia Cloud Platform
```

Comment:
Because of this test failure, I changed `organization:member-remove` => `org:member-remove` or update this complete test file (spaces :) )